### DISCUSSION TITLE
micro-app + react + umi

### DISCUSSION THREAD

**Discussion by majiorxiu** [jd-opensource/micro-app#1019] [Q&A]
基座和子应用都是react+ umi 框架，在基座中打开子应用页面时报出以下错误信息，

![v2-93472ef906383262ecd9ec8a54e1c515_1440w](https://github.com/micro-zoe/micro-app/assets/12382161/93a41c59-0f5d-4c5c-b089-0b5855cda6fe)

![v2-182646c6a868550f3b1412d1cf7ddc10_1440w](https://github.com/micro-zoe/micro-app/assets/12382161/309bbbe3-036c-4ac9-b171-3e1ac5a08337)

点开后是 子应用打包压缩的代码，尝试使用官网提供的关于Cannot read properties of undefined 的错误解决方案，依旧不成功，求大佬帮助
补充一下，子应用单独访问可以正常打开，
![image](https://github.com/micro-zoe/micro-app/assets/12382161/e5648185-2087-405e-88c2-45bc4591c8bf)
这是测试环境正常访问后的资源加载
![image](https://github.com/micro-zoe/micro-app/assets/12382161/88026c41-2061-4411-8a29-07a94aad1b1d)
这是正式环境访问后的资源，
跟朋友讨论也可能是正式环境打包的问题，但问过项目搭建的同事，应该是没有差异的

**Comment by bailicangdu** [comment]
这样看不出问题，能提供一个demo吗

**Reply by majiorxiu** [reply]
是基座的demo吗？

**Reply by bailicangdu** [reply]
基座+子应用完整的复线demo

**Reply by majiorxiu** [reply]
项目本地 运行 依赖有些特殊， 我删减不了

**Reply by bailicangdu** [reply]
只看报错无法确认问题出在哪里

**Reply by majiorxiu** [reply]
这个我明白，但是项目环境我不知道该怎么删减成独立demo，或者您方便远程看看嘛？ 不过现在的情况是本地运行能正常打开子应用，但打包后线上 报这个错

**Reply by bailicangdu** [reply]
有线上地址吗

**Reply by majiorxiu** [reply]
不好意思，这两天才发布上线，能加您社交好友吗？方便把测试账号密码发给您
